http-cydia.radare.org/Dockerfile

```diff
@@ -1,24 +1,3 @@
-FROM ubuntu
+FROM nginx:alpine
 
-ADD www /www
-
-RUN apt-get update && \
-  apt-get install -y nginx make reprepro php5-cli && \
-  rm -rf /var/lib/apt/lists/* && \
-  echo "\ndaemon off;" >> /etc/nginx/nginx.conf && \
-  chown -R www-data:www-data /var/lib/nginx && \
-  apt-get clean
-
-# Requires www/debs to be filled with deb files
-
-# create pkg database
-RUN sh -c "cd /www ; \
-rm -rf /usr/share/nginx/html ; \
-ln -fs /www /usr/share/nginx/html ; \
-make PHP=php5 "
```

Comment:
the idea is that this docker img should be able to autogenerate the repo

-  apt-get install -y nginx make reprepro php5-cli && \